### ISSUE TITLE
Some Other MLLMs Finetuning

### ISSUE THREAD

**Issue by superdabuniu** [2U1/Phi3-Vision-Finetune#36]
Hi, Can you write a tutorial about PaliGemma or LLaVA-NeXT or OneVision Finetuning?

**Comment by 2U1** [comment]
I made the codes where there were no fine-tuning code or need to use some other frameworks.
Using the official repo for LLaVA-Next would be just okay I think.
For the Paligemma, I'll give it a try.

**Comment by superdabuniu** [comment]
hello, Can you improve the logger information in each module during training? Meanwhile the developer can open the log file to debug and analysis the training loss in differnet checkpoints.
